nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/resources/FlowConfiguration.xsd

```diff
@@ -434,7 +434,7 @@
 
     <xs:simpleType name="DataSize">
         <xs:restriction base="xs:string">
-            <xs:pattern value="\d+\s*(B|KB|MB|GB|TB|b|kb|mb|gb|tb)" />
+            <xs:pattern value="\d+(\.\d+)?\s*(B|KB|MB|GB|TB|b|kb|mb|gb|tb)" />
```

Comment:
This doesn't exactly match the regex in the validator: `(\d+(?:\.\d+)?)\s*(B|KB|MB|GB|TB)`, do we need the lookahead in the regex (meaning should they match exactly)?

Reply:
I didn't find a way to have the lookahead accepted. Not sure this is supported in XSD. It was showing me some warnings when I tried in my IDE.